src/pages/graphql/schema/customer/queries/customer.md

```diff
@@ -716,6 +716,164 @@ The following query returns the customer's wish lists. Adobe Commerce allows cus
 }
 ```
 
+### List a company user's purchase orders
+
+The following example returns information about all purchase orders the company user has made since January 1, 2023.
+
+[Purchase orders](../../b2b/purchase-order/) contains additional examples that query purchase order functionality.
+
+**Request:**
+
+```graphql
+{
+    customer {
+        purchase_orders(
+            currentPage: 1,
+            pageSize: 20,
+            filter: {
+                created_date: {
+                  from: "2023-01-21 00:00:00"
+            }
+      }
+        ) {
+            total_count
+            page_info {
+                current_page
+                page_size
+                total_pages
+            }
+            items {
+                uid
+                number
+              order {
+                id
+              }
+              created_at
+              updated_at
+              created_by {
+                firstname
+                lastname
+              }
+              status
+              order {
+                total {
+                  grand_total {
+                    value
+                    currency
+                  }
+                }
+              }
+            }
+        }
+    }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "customer": {
+      "purchase_orders": {
+        "total_count": 2,
+        "page_info": {
+          "current_page": 1,
+          "page_size": 20,
+          "total_pages": 1
+        },
+        "items": [
+          {
+            "uid": "MTE=",
+            "number": "000000011",
+            "order": null,
+            "created_at": "2023-02-01 18:01:40",
+            "updated_at": "2023-02-01 18:01:40",
+            "created_by": {
+              "firstname": "John",
+              "lastname": "Doe"
+            },
+            "status": "PENDING"
+          },
+          {
+            "uid": "MTA=",
+            "number": "000000010",
+            "order": null,
+            "created_at": "2023-01-31 20:46:16",
+            "updated_at": "2023-01-31 21:10:38",
+            "created_by": {
+              "firstname": "John",
+              "lastname": "Doe"
+            },
+            "status": "REJECTED"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### List purchase order approval rules
+
+The following example returns the list of purchase order approval rules.
+
+[Purchase order approval rules](../../b2b/purchase-order-rule/) contains additional examples that query approval rule functionality.
```

Comment:
I see you are referring to linked pages as singular even when their linked text is plural. I'm fine with it, if this is the standard, just pointing it out.